File: vid-webpack-master/src/app/shared/models/subscriber.ts
export class Subscriber {
  id: string;
  name: string;
  isPermitted: boolean;

  constructor(subscriberResponse){
    this.id = subscriberResponse['global-customer-id'];
    this.name = subscriberResponse['subscriber-name'];
    this.isPermitted = subscriberResponse['is-permitted'];
  }
}

File: vid-automation/src/main/java/org/opencomp/simulator/presetGenerator/presets/model/Subscriber.java
package org.opencomp.simulator.presetGenerator.presets.model;


import com.fasterxml.jackson.annotation.JsonProperty;

public class Subscriber {

    @JsonProperty("global-customer-id")
    public String globalCustomerId;

    @JsonProperty("subscriber-name")
    public String subscriberName;

    @JsonProperty("subscriber-type")
    public String subscriberType;

    @JsonProperty("resource-version")
    public String resourceVersion;
}
